music_this_week_app/templates/music_this_week/setup.html

```diff
@@ -12,23 +12,75 @@
         <p><a href="{{ profile }}" target="_blank">{{ username }}</a>&nbsp; &nbsp;&nbsp;<a href="/login/?retry=true">Switch Users</a></p>
         <p><span style="text-decoration: underline;">Search Arguments:</span></p>
         <form action="/search" id="form">
-            <p><strong>Location:</strong> <input id = "loc_field" type="text" name="location" value="San Francisco"></p>
+            <p id = 'location_options'><strong>Location:</strong> <input id = "loc_field" type="text" name="location" value="{{location}}"> 
+               <input type = "checkbox" id = "latlng_checkbox" name = "latlng_checkbox" onchange = "clickCheckBox.call(this)"> Use my current location </input></p>
             <p><strong>Time range:</strong> <input id = "date_field" type="text" name="date" value="Next 7 Days"></p>
+            <p><input id = "hidden_lat" hidden = true name = "hidden_lat"></p>
+            <p><input id = "hidden_lon" hidden = true name = "hidden_lon"></p>
             <input hidden=true name="nResults" value="200">
             <button type="submit" id="search-button" onclick="buttonClick(this.form)">Search</button>
         </form>
         <p>&nbsp;</p>
         <p id="wait-text" hidden="True">Please be patient. In few minutes a playlist called musicThisWeek will appear in your spotify app.<br><br>If you close this page, the playlist will still be created in the background.</p>
     </div>
-
     <script>
+        //init a variable that will contain user lat lng if checkbox clicked
```

Comment:
Does this boolean variable contain user lat lng if checkbox clicked? That's what the comment implies